pkg/task/cluster.go

```diff
@@ -27,11 +28,31 @@ func (ti *TaskInterface) ProcessClusters(directory string) error {
 			return err
 		}
 	}
+	if cluster == nil {
```

Comment:
It doesn't look like `retrieveClusterFromEnvironment` can ever return a nil value, can this check go away?